src/Symfony/Component/HttpKernel/Tests/HttpCache/ResponseCacheStrategyTest.php

```diff
@@ -370,13 +370,29 @@ public function cacheControlMergingProvider()
         ];
 
         yield 'merge max-age and s-maxage' => [
-            ['public' => true, 's-maxage' => '60', 'max-age' => null],
+            ['public' => true, 'max-age' => '60'],
```

Comment:
Short explanation what happens here:

Previously, not all responses had `max-age`, so it was dropped. But all responses were assumed to have `s-maxage` (because that can be substituted by `maxage`), so it was kept.

With this change, the `s-maxage` is computed as 60, because we have the explicit `s-maxage` and `max-age=60`. And we get `max-age=60`, because one response sets that explicitly, whereas the other does not care while being `public`. So the result is `s-maxage=60, maxage=60`, which is being merged into a single `max-age=60`.